tests/e2e/auth/acceptprojectinvitation/acceptprojectinvitation-page.test.ts

```diff
@@ -0,0 +1,43 @@
+import { expect } from "@playwright/test";
+import { test } from "./acceptprojectinvitation-page-fixture";
+
+test.use({ user: null });
+
+test.describe("Accept Project Invitation Verification Navigation", () => {
```

Comment:
```suggestion
test.describe("Accept Project Invitation Navigation", () => {
```
Otherwise this sounds very complicated.